### ISSUE TITLE
Pressing 't' in search bar escapes the search bar field

### ISSUE THREAD

**Issue by codeworkplsthx** [minbrowser/min#710]
Haven't seen this posted, which surprises me because it would seem to be common? 
Click on the search bar, press 't'. It cancels input and escapes the text field. 

For reference:
[t_button_bug.webm.zip](https://github.com/minbrowser/min/files/2858449/t_button_bug.webm.zip)

**Comment by PalmerAL** [comment]
What keyboard layout are you using? Do you have "t" set as a shortcut for anything in the keyboard shortcuts section of the preferences (or did you make any other changes to those settings)?

**Comment by codeworkplsthx** [comment]
> What keyboard layout are you using?
 
My keyboard layout is English > U.S. 

> Do you have "t" set as a shortcut for anything in the keyboard shortcuts section of the preferences?

Found the culprit.
![app_settings](https://user-images.githubusercontent.com/5252860/52720057-d23c0d80-2f9e-11e9-9284-e97113178631.png)

My mistake. Sorry about that.

**Comment by PalmerAL** [comment]
That looks like a bug actually; with that shortcut, I'm guessing that nothing should happen unless you press both "g" and "t" at the same time.

**Comment by codeworkplsthx** [comment]
It appears that shortcuts consisting only of letters "respond" only when User is typing inside search bar. For example, when User is typing inside search bar, "g+t" exits the search bar; Program responds to "g+t", but not correctly. However, when User is not typing inside search bar, Program does not respond to "g+t" at all. This applies in the case of "a+b" as well.

On the other hand, Program responds to "shift+t" correctly regardless of User's state. 

So to sum up, Program behaves correctly if shortcut has form "<meta_key(s)>+<alpha_num>".

**Comment by PalmerAL** [comment]
That sounds accurate; we have two different ways of handling keyboard shortuts depending on whether the keyboard focus is in the browser UI or on a webpage, and neither one appears to work correctly with this:

* The [Mousetrap](https://github.com/ccampbell/mousetrap) library, which is used for the browser UI, treats "g+t", as just being "t", so pressing "g" doesn't do anything, and pressing "t" on its own runs the shortcut.
* [keybindings.js](https://github.com/minbrowser/min/blob/master/js/keybindings.js#L462), for a webpage, only matches one letter per shortcut, so anything with more than one letter won't work at all like you noted.

**Comment by Syndamia** [comment]
It seems that shortcuts of type `<alpha_num>+<alpha_num>` don't work anymore. 

I did some testing and shortcuts of type `<alpha_num>` and `<mata_key(s)>+<alpha_num>` work as expected, but of type `<alpha_num>+<alpha_num>` don't do anything.

@PalmerAL do you know anything about this?
